tests/Unit/Framework/TestHelpers.hpp

```diff
@@ -229,6 +229,30 @@ void test_reverse_iterators(Container& c) {
   }
 }
 
+/*!
```

Comment:
After you've dropped the other 2 commits and switched to `if constexpr` this commit is also no longer needed.

bonus: the PR will be super easy to review! :D